src/components/ProgramRecord/ProgramRecord.scss

```diff
@@ -1,20 +1,23 @@
 .program-record-wrapper {
-    @extend .my-4\.5;
+    margin-top: calc(var(--pgn-spacing-spacer-base) * 2) !important;
 
     .back-to-records:hover {
         background: none;
     }
 
     .program-record-nav {
-        @extend .d-flex, .justify-content-between, .mb-3;
+        display: flex !important;
+        justify-content: space-between !important;
+        margin-bottom: var(--pgn-spacing-spacer-base) !important;
 
         .program-record-actions {
-            @extend .d-flex;
+            display: flex !important;
             gap: 0.5rem;
         }
 
         .download-record {
-            @extend .d-flex, .flex-row-reverse;
+            display: flex !important;
+            flex-direction: row-reverse !important;
```

Comment:
Based on https://github.com/openedx/frontend-app-authn/pull/1436#discussion_r2079869474 moving away from `@extend` in this way is good as-is!